### PR TITLE
Prepared storage for GA release of STG77

### PR DESCRIPTION
**Pull request by gapra-msft**

### CODE REVIEW

File: sdk/communication/pom.xml [hunk 1]
```diff
@@ -29,22 +29,22 @@
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-communication-common</artifactId>
-          <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+          <version>1.0.2</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```
Comment (gapra-msft) [inline sdk/communication/pom.xml:32]: I'm not sure why this got updated. I will revert this

**Review by kasobol-msft — APPROVED** [review]
LGTM. Please make sure nightly and weekly pipeline is happy.

**Pull request merged**